forEach.js

```diff
@@ -11,3 +11,44 @@ function cb(item, index, arr){
 };
 
 customForEach(family, cb);
+
+// Slice
+const array = ["t", "e", "s", "r"];
+
+function slice(arr,start,end) {
+  end = end || arr.length;
```

Comment:
Please try not to change the incoming arguments, create a new variable instead
```suggestion
  const finish = end || arr.length;
```